INSTALLING.md

```diff
@@ -24,10 +24,16 @@ If you are running a distribution which does not match exactly a repository uplo
 
 If you are running LinuxMint 17.1 Rebecca, which is downstream of Ubuntu Trusty and Debian Jessie, you can run:
 
-`curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | os=debian dist=jessie sudo -E sudo bash`
+`curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | os=debian dist=jessie sudo -E bash`
```

Comment:
This looks obviously correct. This is now obsolete, though, but we don't have to change that now.

Reply:
Agreed, didn't want to go changing the entire page though